### ISSUE TITLE
Could not cast value of type 'Swift.Int64' (0x10a3a9f80) to 'Swift.String' (0x10a3b04d8).

### ISSUE THREAD

**Issue by etayluz** [realm/SwiftLint#2197]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Rule Request

If this is a bug report, please ignore this section.

If this is a new rule request, please ignore all sections below this one, format
this issue's title as `Rule Request: [Rule Name]` and describe:

1. Why should this rule be added? Share links to existing discussion about what
   the community thinks about this.
2. Provide several examples of what _would_ and _wouldn't_ trigger violations.
3. Should the rule be configurable, if so what parameters should be configurable?
4. Should the rule be opt-in or enabled by default? Why?
   See [README.md](../README.md#opt-in-rules) for guidelines on when to mark a
   rule as opt-in.

### Bug Report
I just added SwiftLint latest release to my project via CocoaPods.
When I run it I get the error: Could not cast value of type 'Swift.Int64' (0x10a3a9f80) to 'Swift.String' (0x10a3b04d8).
If I tell swiftlint to ignore the offending file - it happens on every file and exits.

What am I doing wrong here?

##### Complete output when running SwiftLint, including the stack trace and command used
>swiftlint
Loading configuration from '.swiftlint.yml'
Linting Swift files in current working directory
Linting 'Event.swift' (1/24)
Could not cast value of type 'Swift.Int64' (0x10a3a9f80) to 'Swift.String' (0x10a3b04d8).
Abort trap: 6
```bash
$ swiftlint lint
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 0.11.1
* Installation method used (Homebrew, CocoaPods, building from source, etc)? CocoaPods
* Paste your configuration file:

```yml
excluded: # paths to ignore during linting. Takes precedence over `included`.
  - Pods
  - Resources/GlobalConstants.swift
force_cast: warning # implicitly
force_try:
  severity: warning # explicitly
line_length: 145
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)? No
  If so, paste their relative paths and respective contents.
* Which Xcode version are you using (check `xcode-select -p`)? 9.3
* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
// This triggers a violation:
let foo = try! bar()
```

**Comment by etayluz** [comment]
Turns out I wasn't on the latest (like I thought i was):
>swiftlint version
0.25.1

Uninstall swiftlint (global) and reinstall:
https://stackoverflow.com/questions/41865499/how-to-install-a-specific-version-of-swiftlint

**Comment by jonasman** [comment]
still happens on 0.28.2

**Comment by jpsim** [comment]
Please open a new issue with steps to reproduce.
